app/controllers/api/v1/short_form_controller.rb

```diff
@@ -33,6 +33,11 @@ def delete_proof
     end
   end
 
+  def submit_application
+    response = SalesforceService.submit_application(params)
```

Comment:
@chrisdolendo this really should send something like `application_params` which performs some checks e.g. `require(:application)`, `permit(...)` rather than sending through the entire `params` object without any checks. For example this object includes things like `controller` and `action` which end up getting sent along as well. It will be slightly annoying in the sense that we will want to double check all of the permitted field names inside of the Rails controller much like we're already doing in JS, but it's somewhat necessary.